dist/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/c6f38472f5.js" crossorigin="anonymous"></script>
+    <title>To Do List</title>
+<script defer src="runtime.bundle.js"></script><script defer src="main.bundle.js"></script></head>
+<body>
+    <div class="tasks">
+        <h2>Today's To Do<i class="fa-solid fa-rotate fa-rotate-270 fa-xs display-rotate"></i></h2>
```

Comment:
```suggestion
    <section class="tasks">
        <h1>Today's To Do<i class="fa-solid fa-rotate fa-rotate-270 fa-xs display-rotate"></i></h1>
    </section >
```

Please Use HTML5 semantic tag for better SEO and screen reader optimation